yb-voyager/cmd/exportData.go

```diff
@@ -264,6 +277,39 @@ func exportPGSnapshotWithPGdump(ctx context.Context, cancel context.CancelFunc,
 	return nil
 }
 
+func getPGDumpSequencesAndValues() (map[*sqlname.SourceName]int64, error) {
+	sequenceValueMap := map[*sqlname.SourceName]int64{}
+	sequenceFilePath := filepath.Join(exportDir, "data", "postdata.sql")
+	seqeuencesTextFileDataBytes, err := os.ReadFile(sequenceFilePath)
+	if err != nil {
+		utils.ErrExit("read file %q: %v", sequenceFilePath, err)
+	}
+
+	sequencesTextFileData := strings.Split(string(seqeuencesTextFileDataBytes), "\n")
+	numLines := len(sequencesTextFileData)
+	setvalRegex := regexp.MustCompile(`(?i)SELECT.*setval\((?P<args>.*)\)`)
+
+	for i := 0; i < numLines; i++ {
+		matches := setvalRegex.FindStringSubmatch(sequencesTextFileData[i])
+		if len(matches) > 0 {
+			matchIndex := setvalRegex.SubexpIndex("args")
+			if matchIndex > len(matches) {
+				return nil, fmt.Errorf("invalid index %d for matches - %s for line %s", matchIndex, matches, sequencesTextFileData[i])
+			}
+			setValArgs := matches[matchIndex]
+			setValArgsArr := strings.Split(setValArgs, ",")
+			seqVal, err := strconv.ParseInt(strings.TrimSpace(setValArgsArr[1]), 10, 64)
+			if err != nil {
+				return nil, fmt.Errorf("parse %s to int in line - %s: %v", setValArgsArr[1], sequencesTextFileData[i], err)
+			}
+			seqNameRaw := setValArgsArr[0][1 : len(setValArgsArr[0])-1]
+			seqName := sqlname.NewSourceNameFromQualifiedName(seqNameRaw)
+			sequenceValueMap[seqName] = seqVal
+		}
+	}
+	return sequenceValueMap, nil
+}
```

Comment:
Sometimes using smaller+idiomatic names can lead to more readable code. The name of the function itself provides some context for the names inside its scope. The context can be utilised to reduce the local name lengths.
```
func getPGDumpSequencesAndValues() (map[*sqlname.SourceName]int64, error) {
	result := map[*sqlname.SourceName]int64{}
	path := filepath.Join(exportDir, "data", "postdata.sql")
	bs, err := os.ReadFile(path)
	if err != nil {
		utils.ErrExit("read file %q: %v", path, err)
	}
	lines := strings.Split(string(bs), "\n")

    // Sample line: SELECT pg_catalog.setval('public.actor_actor_id_seq', 200, true);
	re := regexp.MustCompile(`(?i)SELECT.*setval\((?P<args>.*)\)`)

	for _, line := range lines {
		matches := re.FindStringSubmatch(line)
		if len(matches) == 0 {
			continue
		}
		idx := re.SubexpIndex("args")
		if idx > len(matches) {
			return nil, fmt.Errorf("invalid index %d for matches - %s for line %s", idx, matches, lines[i])
		}
		match := matches[idx]
		args := strings.Split(match, ",")

		seqNameRaw, _ := strconv.Unquote(args[0])
		seqName := sqlname.NewSourceNameFromQualifiedName(seqNameRaw)

		seqVal, err := strconv.ParseInt(strings.TrimSpace(args[1]), 10, 64)
		if err != nil {
			return nil, fmt.Errorf("parse %s to int in line - %s: %v", args[1], lines[i], err)
		}
		result[seqName] = seqVal
	}
	return result, nil
}
```